src/screens/authentication/UsernameRequest/index.js

```diff
@@ -33,14 +34,15 @@ const Error = ({ error }) => (
 const SubmitButton = ({ disabled, onPress }) => {
   if (disabled) {
     return (
-      <GrayButton>
+      <GrayButton style={styles.submitButton}>
         <Description white>Continuar</Description>
       </GrayButton>
     )
   } else {
     return (
       <PrimaryButton
         onPress={onPress}
+        style={styles.submitButton}
       >
         <Description white>Continuar</Description>
       </PrimaryButton>
```

Comment:
This is old code from before we added the `disabled` prop into `PrimaryButton`.

Please change it to use the prop. It would be similiar to this

```
const SubmitButton = ({ disabled, onPress }) => {
 return (
  <PrimaryButton
   onPress={onPress}
   style={styles.submitButton}
   disabled={disabled}
  >
   <Description white>Continuar</Description>
  </PrimaryButton>
 )
}
```